voxel51/utils.py

```diff
@@ -147,7 +147,7 @@ def from_string(cls, s):
         Returns:
             an instance of the Serializable subclass
         '''
-        return cls.from_dict(json.loads(s))
+        return cls.from_dict(json.loads(str(s)))
```

Comment:
Why is `str` okay here, rather than something like your `_load_json` function in `eta`?

Reply:
Good point. I updated to use the solution from ETA. Thanks.